testsuite/tests/conftest.py

```diff
@@ -51,6 +53,13 @@ def term_handler():
     signal.signal(signal.SIGTERM, orig)
 
 
+@pytest.fixture(scope="session")
+def api_client_type(pytestconfig):
+    """Do tests use REST API client(False) or CR Capabilities client(True)?"""
```

Comment:
I would rename this fixture to something like `use_capabilities_client` to match bool return value and for more clear usage later in the code.